wandb/plots/precision_recall.py

```diff
@@ -0,0 +1,92 @@
+import wandb
+from wandb import util
+from wandb.plots.utils import test_missing, test_types, encode_labels
+np = util.get_module("numpy", required="Logging plots requires numpy")
+scikit = util.get_module("sklearn")
+chart_limit = wandb.Table.MAX_ROWS
```

Comment:
Optional deps should go in the func.  Also, if scikit is required we should make it so.